RadixWallet/Features/NewConnectionFeature/Coordinator/NewConnection+Reducer.swift

```diff
@@ -215,6 +215,12 @@ public struct NewConnection: Sendable, FeatureReducer {
 
 		case let .root(.scanQR(.delegate(.scanned(qrString)))):
 			return .run { send in
+				if let _ = try? HexCodable32Bytes(hex: qrString) {
```

Comment:
just FYI after you merge in my monster PR into this, the `HexCodable32Bytes` is called `Exactly32Bytes`.